src/main/config/config.c

```diff
@@ -201,6 +202,14 @@ static void validateAndFixRatesSettings(void)
     }
 }
 
+static void validateAndFixPositionConfig(void)
+{
+    if (positionConfig()->altNumSatsBaroFallback >= positionConfig()->altNumSatsGpsUse) {
+        positionConfigMutable()->altNumSatsGpsUse = POSITION_DEFAULT_ALT_NUM_SATS_GPS_USE;
+        positionConfigMutable()->altNumSatsBaroFallback = POSITION_DEFAULT_ALT_NUM_SATS_BARO_FALLBACK;
+    }
```

Comment:
```suggestion
 if (positionConfig()->altNumSatsBaroFallback >= positionConfig()->altNumSatsGpsUse) {
        positionConfigMutable()->altNumSatsBaroFallback = positionConfigMutable()->altNumSatsGpsUse - 1;
 }
```

Reply:
I've discussed this before. Maybe it is dismissed and you haven't saw it.
> With this change, if goodGpsSats is 4, and badGpsSats is 5, for example, will end with badGpsSats with a value of 3, that is outside the minimum for this variable.
I think is better to reset both to our default values and simplifies all.
I think remember other variables with similar problem have a "special" method that validates all the values, I will try to find the method in the code, to make it the same.

It's safer simply reset the values with problems to good defaults.

Reply:
Sorry, missed that conversation.
I don't see problem with that case. With 4 sats you get altitude, with 3 you don't. IMO this works as expected and is desired.
Also, altNumSatsBaroFallback minimum shall be 3 for this reason, otherwise you can't get altitude with 4 sats. (with 'hysteresis' implementation, this would be obvious)

Reply:
You don't know what is the parameter that the user put wrong, so modify one in a blind way can be bad and let both with wrong values. Better to let both with safe defaults.

Reply:
Ok